docs/_posts/2017-01-18-jekyll-3-4-0-released.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Jekyll turns 3.4.0'
+date: 2017-01-18 14:19:13 -0500
+author: parkr
+version: 3.4.0
+categories: [release]
+---
+
+Hey there! We have a quick update of Jekyll for you to enjoy this January.
+Packed full of bug fixes as usual, thanks to the tireless efforts of our
+exceptional Jekyll community. Three changes to call out:
+
+1. If you're a big fan of [`where_by_exp`](/docs/filters/), you'll be an
+even bigger fan of [`group_by_exp`](/docs/filters/).
+2. Using a custom timezone in Jekyll on Windows? Yeah, sorry that hasn't ever worked
+   properly. We made it possible to accurately [set the timezone using IANA
+   timezone codes](https://jekyllrb.com/docs/windows/#timezone-management).
+3.
```

Comment:
Better theme documentation I guess. New include section. New Circle CI section, etc.
Docs have received a lot of contributions since last version. Mainly thanks to @tomjohnson1492